src/GenAPI/Microsoft.DotNet.GenAPI/INamedTypeSymbolExtension.cs

```diff
@@ -159,5 +160,56 @@ public static bool IsBoundGenericType(this INamedTypeSymbol namedType)
 
             return true;
         }
+
+        // Synthesize an internal default constructor for the type with implicit default constructor and the base type without.
```

Comment:
As this API is publicly exposed, please use triple-slash documentation here: ///

Reply:
Updated

```C#
/// <summary>
/// Synthesize an internal default constructor for the type with implicit default constructor and the base type without.
/// </summary>
/// <param name="namedType">A loaded named type symbol <see cref="INamedTypeSymbol"/>.</param>
/// <param name="symbolFilter">Assembly symbol filter <see cref="ISymbolFilter"/>.</param>
```